modules/fieldlinessequence/rendering/renderablefieldlinessequence.cpp

```diff
@@ -871,134 +762,107 @@ void RenderableFieldlinesSequence::extractTriggerTimesFromFileNames() {
+            _extraVars,
             extraMagVars
         );
 
         if (isSuccessful) {
             addStateToSequence(newState);
-            if (!outputFolder.empty()) {
-                newState.saveStateToOsfls(outputFolder);
+            if (!_outputFolderPath.empty()) {
+                newState.saveStateToOsfls(_outputFolderPath);
             }
         }
     }
     return true;
 }
 
-/*
-* Returns false if it fails to extract mandatory information!
-*/
-bool RenderableFieldlinesSequence::extractCdfInfoFromDictionary(std::string& seedFilePath,
-                                                                std::string& tracingVar,
-                                                      std::vector<std::string>& extraVars)
+std::unordered_map<std::string, std::vector<glm::vec3>>
+    extractSeedPointsFromFiles(std::filesystem::path filePath)
 {
-    if (_dictionary->hasValue<std::string>(KeyCdfSeedPointFile)) {
-        seedFilePath = _dictionary->value<std::string>(KeyCdfSeedPointFile);
-        if (std::filesystem::is_regular_file(seedFilePath)) {
-            seedFilePath = absPath(seedFilePath).string();
-        }
-        else {
-            LERROR(fmt::format(
-                "{}: The specified seed poitn file: '{}' does not exist",
-                _identifier, seedFilePath
-            ));
-            return false;
-        }
-    }
-    else {
-        LERROR(fmt::format("{}: Must specify '{}'", _identifier, KeyCdfSeedPointFile));
-        return false;
-    }
+    std::vector<std::string> files;
+    std::unordered_map<std::string, std::vector<glm::vec3>> outMap;
 
-    if (_dictionary->hasValue<std::string>(KeyCdfTracingVariable)) {
-        tracingVar = _dictionary->value<std::string>(KeyCdfTracingVariable);
-    }
-    else {
-        tracingVar = "b"; //  Magnetic field variable as default
-        LWARNING(fmt::format(
-            "{}: No '{}', using default '{}'",
-            _identifier, KeyCdfTracingVariable, tracingVar
+    if (!std::filesystem::is_directory(filePath)) {
+        LERROR(fmt::format(
+            "The specified seed point directory: '{}' does not exist", filePath
         ));
+        return outMap;
     }
 
-    if (_dictionary->hasValue<ghoul::Dictionary>(KeyCdfExtraVariables)) {
-        ghoul::Dictionary extraQuantityNamesDictionary =
-            _dictionary->value<ghoul::Dictionary>(KeyCdfExtraVariables);
-        const size_t nProvidedExtras = extraQuantityNamesDictionary.size();
-        for (size_t i = 1; i <= nProvidedExtras; ++i) {
-            extraVars.push_back(
-                extraQuantityNamesDictionary.value<std::string>(std::to_string(i))
-            );
+    namespace fs = std::filesystem;
+    for (const fs::directory_entry& spFile : fs::directory_iterator(filePath)) {
+        std::string seedFilePath = spFile.path().string();
+        if (!spFile.is_regular_file() ||
+            seedFilePath.substr(seedFilePath.find_last_of('.')+1) != "txt")
```

Comment:
Needs some air around the plus sign, for readability
`seedFilePath.substr(seedFilePath.find_last_of('.') + 1) != "txt")`